pkg/continuoustest/client.go

```diff
@@ -207,13 +209,48 @@ func (c *Client) sendWriteRequest(ctx context.Context, req *prompb.WriteRequest)
 	return httpResp.StatusCode, nil
 }
 
+// RequestOption defined a functional-style request option.
+type RequestOption func(options *requestOptions)
```

Comment:
Doesn't `RequestOptions` also have to be exported, if a client outside of the package uses (the exported) `RequestOption`?
```suggestion
type RequestOption func(options *RequestOptions)
```

Reply:
It's not required. Request options are configured using `WithXXX()` functions, but the actual options struct doesn't have to be exported (and they're typically not exported).